tests/unit/test_simulator.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from model import storage
+from model.simulator import METHODS, simulate
+
+
+def test_tmy_result():
+    tmy_objectives = ['bigg:akg']
```

Comment:
I think it would be cool to parametrize these tests, i.e.,
```python
@pytest.mark.parametrize("objective", ['bigg:akg'])
def test_tmy_result(objective):
```